### PR TITLE
MWPW-130838 remove the offer id from the checkout link

### PR DESCRIPTION
**Pull request by VKniaz**
Remove the offer id from the checkout link.

Resolves: [MWPW-130838](https://jira.corp.adobe.com/browse/MWPW-130835)

- Before: https://main--milo--adobecom.hlx.page/drafts/vkniaziev/document3?martech=off
- After: https://MWPW-130838--milo--adobecom.hlx.page/drafts/vkniaziev/document3?martech=off

### CODE REVIEW

File: libs/blocks/ost/ost.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ const IMS_COMMERCE_CLIENT_ID = 'aos_milo_commerce';
 const IMS_PROD_URL = 'https://auth.services.adobe.com/imslib/imslib.min.js';
 const OST_VERSION = '1.11.0';
 const OST_BASE = `https://www.stage.adobe.com/special/tacocat/ost/lib/${OST_VERSION}`;
-const OST_SCRIPT_URL = `${OST_BASE}/index.js`;
+const OST_SCRIPT_URL = 'http://localhost:3000/libs/blocks/ost/index.js';
```
Comment (honstar) [inline libs/blocks/ost/ost.js:9]: This can't be right and is probably a left-over from testing.

**Review by honstar — CHANGES_REQUESTED** [review]

**Comment by honstar** [comment]
Also, I already don't see the `offerid` request parameter in the `before` page markup: view-source:https://main--milo--adobecom.hlx.page/drafts/vkniaziev/document3?martech=off

File: libs/blocks/ost/ost.js (hunk 1 above)
Comment (VKniaz) [reply libs/blocks/ost/ost.js:9]: thanks

**Comment by VKniaz** [comment]
> Also, I already don't see the `offerid` request parameter in the `before` page markup: `view-source:https://main--milo--adobecom.hlx.page/drafts/vkniaziev/document3?martech=off`.

this is correct because branch doesn't affect the link that already was pasted.

**Review by vladen — APPROVED** [review]

**Review by overmyheadandbody — COMMENTED** [review]
A few checks are failing. You can add the 'needs-verification' label to the PR. You then need to check unit tests and PSI concerns to ensure previous performance isn't being affected.

**Review by chrischrischris — APPROVED** [review]

**Comment by honstar** [comment]
@VKniaz : can you please fix the eslint errors?

**Comment by honstar** [comment]
@VKniaz : what's the status on this PR? Can you please address the change requests? Also, why was the `do not merge` label added to this PR?

**Comment by chrischrischris** [comment]
Closing PR due to inactivity - feel free to re-open when it's ready.

**Pull request closed without merging**